templates/404.html

```diff
@@ -3,9 +3,9 @@
 <html lang={{ config.lang_code }} dir={{ lang_dir }}>
   <head>
     {% block head_extras %}{% endblock %}
-    <title>Citation Hunt</title>
+    <title>{{ config.strings.tooltitle }}</title>
     <meta name="robots" content="nofollow, noodp, noindex">
-    <meta name="description" content="Citation Hunt - {{ config.strings.introduction }}">
+    <meta name="description" content="{{ config.strings.tooltitle }} - {{ config.strings.introduction }}">
```

Comment:
@ricordisamoa I am aware of that guideline, but IMO this is not such a case. The order would be the same in any language I know of; title first, description after (but if you know any counter-examples, I'd love to hear about them!). Making the separator translatable I can agree with, but is it really necessary? It's just in a Meta tag, which isn't exactly the most prominent place.